nf_core/pipelines/lint/local_component_structure.py

```diff
@@ -0,0 +1,31 @@
+import logging
+from pathlib import Path
+
+log = logging.getLogger(__name__)
+
+
+def local_component_structure(self):
+    """
+    Check that the local modules and subworkflows directories in a pipeline have the correct format:
+
+    .. code-block:: bash
+
+        modules/local/TOOL/SUBTOOL
+
+    Prior to nf-core/tools release 3.0.3 the directory structure allowed top-level `*.nf` files:
```

Comment:
I think next one will be 3.1.0 :) 
```suggestion
    Prior to nf-core/tools release 3.1.0 the directory structure allowed top-level `*.nf` files:
```